addons/agent_mirrorpeer_controller.go

```diff
@@ -430,7 +431,7 @@ func (r *MirrorPeerReconciler) toggleMirroring(ctx context.Context, storageClust
 			return fmt.Errorf("StorageCluster %q does not have required PeerSecrets", storageClusterName)
 		}
 	} else {
-		sc.Spec.Mirroring = nil
+		sc.Spec.Mirroring.Enabled = false
```

Comment:
We should set it to nil as we will no longer be managing this spec once mirroring is disabled.